apps/settings/js/panels/achievements/panel.js

```diff
@@ -0,0 +1,19 @@
+define(function(require) {
+  'use strict';
+
+  var SettingsPanel = require('modules/settings_panel');
+  var AchievementsList = require('panels/achievements/achievements_list');
+  var SettingsListener = require('shared/settings_listener');
+
+  return function ctor_achievements_panel() {
+    var achievementsList;
+
+    return SettingsPanel({
+      onInit: function(panel) {
+        achievementsList = AchievementsList(panel);
+        SettingsListener.observe('achievements', achievementsList.achievements,
```

Comment:
The other thing is that it would be better to observe the change only when users are in the panel, so let's do it in `onBeforeShow` and un-observe in `onHide`.